### PR TITLE
Wallet API: added walletExists logic

### PR DESCRIPTION
**Pull request by Jaqueeee**

### CODE REVIEW

File: src/wallet/api/wallet_manager.cpp [hunk 1]
```diff
@@ -81,6 +81,12 @@ bool WalletManagerImpl::closeWallet(Wallet *wallet)
 
 bool WalletManagerImpl::walletExists(const std::string &path)
 {
+    bool keys_file_exists;
+    bool wallet_file_exists;
+    tools::wallet2::wallet_exists(path, keys_file_exists, wallet_file_exists);
+    if(keys_file_exists){
+        return true;
+    }
```
Comment (moneromooo-monero) [inline src/wallet/api/wallet_manager.cpp]: easier to just return keys_file_exists

File: src/wallet/api/wallet_manager.cpp [hunk 2]
```diff
@@ -91,7 +97,7 @@ std::vector<std::string> WalletManagerImpl::findWallets(const std::string &path)
     boost::filesystem::path work_dir(path);
     // return empty result if path doesn't exist
     if(!boost::filesystem::is_directory(path)){
-      return result;
+        return result;
```
Comment (moneromooo-monero) [inline src/wallet/api/wallet_manager.cpp]: gratuitous whitespace changes are frowned upon (unless the previous state is dire)

File: src/wallet/api/wallet_manager.cpp (hunk 1 above)
Comment (Jaqueeee) [inline src/wallet/api/wallet_manager.cpp]: Right.  You wan't me to rebase with squash or just add a commit? @moneromooo-monero

Comment (moneromooo-monero) [inline src/wallet/api/wallet_manager.cpp]: If you're going to change this, squash. Adding a commit to change trivial stuff in the same PR just adds spam. We aim for easy to review. You don't *have* to change it though, I OK'd it. Though the gratuitous whitespace below would be nice to fix :D

**Pull request merged**